### PR TITLE
fix toplevel string parsing in nested binary ops

### PR DESCRIPTION
**Pull request by pfitzseb**
Fixes https://github.com/julia-vscode/CSTParser.jl/issues/262.

For every PR, please check the following:
- [x] End-user documentation check. If this PR requires end-user documentation in the Julia VS Code extension docs, please add that at https://github.com/julia-vscode/docs.
- [x] Changelog mention. If this PR should be mentioned in the CHANGELOG for the Julia VS Code extension, please open a PR against https://github.com/julia-vscode/julia-vscode/blob/master/CHANGELOG.md with those changes.

### CODE REVIEW

File: src/CSTParser.jl [hunk 1]
```diff
@@ -187,12 +187,12 @@ end
 Used for top-level parsing - attaches documentation (such as this) to expressions.
 """
 function parse_doc(ps::ParseState)
-    if (kindof(ps.nt) === Tokens.STRING || kindof(ps.nt) === Tokens.TRIPLE_STRING) && !isemptyws(ps.nws)
+    if (kindof(ps.nt) === Tokens.STRING || kindof(ps.nt) === Tokens.TRIPLE_STRING) && isnewlinews(ps.nws)
```
Comment (pfitzseb) [inline src/CSTParser.jl:190]: I think the original code was correct:
```
julia> "asd" f(x)=x
f

help?> f
search: f fd for fma fld fld1 fill fdio Foo frexp foldr foldl flush floor float first fill! fetch fldmod filter falses finally foreach

  asd
```

Comment (ZacLN) [reply src/CSTParser.jl:190]: Ah right

Comment (ZacLN) [inline src/CSTParser.jl:190]:
```suggestion
    if (kindof(ps.nt) === Tokens.STRING || kindof(ps.nt) === Tokens.TRIPLE_STRING) && !isemptyws(ps.nws)
```

**Comment by davidanthoff** [comment]
What is the status here? Just needs to be approved and then can ship in the next patch release?

**Comment by pfitzseb** [comment]
Yep. This should be good to go for the next patch.

**Review by davidanthoff — APPROVED** [review]

**Pull request merged**